jsonld/transformer.go

```diff
@@ -0,0 +1,56 @@
+/*
+ * Nuts node
+ * Copyright (C) 2022 Nuts community
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package jsonld
+
+import (
+	"encoding/json"
+
+	"github.com/nuts-foundation/go-did/vc"
+	"github.com/piprate/json-gold/ld"
+)
+
+// JSONLdBase is the base URL used for IRIs that are unknown when expanding a JSON-LD document.
+const JSONLdBase = ""
+
+type transformer struct {
```

Comment:
really feels like a helper type, is it required that it is a type itself or should it just be some util funcs?

Reply:
I was going to abstract the contextLoader. But given the other PRs, the contextLoader is required at different levels. Replaced it with a simple Reader type.